schemas/api_schema.js

```diff
@@ -267,7 +273,8 @@ type Mutation {
                             attendees,
                             platform,
                             platform_media_id,
-                            img_source
+                            img_source,
+                            acive
```

Comment:
typo. Plus, why even allow this to be edited in the first place?

Reply:
because it is the only way to edit it, if a user wants to hide his session or make it visible

typo fixed